src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -111,6 +112,7 @@
 	private boolean sprinting = false;
 	private boolean flying = false;
 	private boolean whitelisted = true;
+	private boolean collides = true;
```

Comment:
Why is this not named `collidable` too?
Seems inconsistent to me.

Reply:
I just took the name that is used in craftbukkit 😀